test/test_autograd.py

```diff
@@ -4185,11 +4185,14 @@ def run_functional_checks(test_case, test_name, name, apply_fn, run_grad_checks,
         test_case.assertEqualTypeString(self_variable, self_variable.grad)
         test_case.assertEqual(self_variable.size(), self_variable.grad.size())
 
+separate_complex_tests = ['log', 'log10', 'log1p', 'log2', 'reciprocal']
```

Comment:
Could you add a comment for what this list contains?
These are the ones that run only for complex?